Server/upload/25a6488a2135bdeae7e26a8e9baac62f

```diff
@@ -1,240 +0,0 @@
-{
-  "reference": {
-    "type": "book",
-    "publisher": "University of California Press",
-    "authors": [
-      {
-        "firstname": "Stanley",
-        "middlename": "P.",
```

Comment:
restore this file it is used in tests 
![image](https://user-images.githubusercontent.com/22376976/107131666-8dee5a00-68a6-11eb-9aa9-7c634a3489f5.png)

Reply:
done